test/scripts/dcLana.test.js

```diff
@@ -0,0 +1,71 @@
+import lanaLogging from '../../acrobat/scripts/dcLana';
+import * as sinon from 'sinon';
+import { expect } from '@esm-bundle/chai';
+
+describe('Test dcLana script', async () => {
+  let clock;
+
+  before(() => {
```

Comment:
What do you think to add console.log in case something goes wrong?
Something like this?
``` 
before(() => {
  sinon.spy(console, 'log');
});

after(() => {
  console.log.restore();
});

Reply:
Is it the standard way in Milo? I can add that.

Reply:
dcLana.js doesn't call `console.log()`. There is no point to spy it.